core/src/main/java/com/sequenceiq/cloudbreak/service/image/CustomImageProvider.java

```diff
@@ -28,6 +28,7 @@ public StatedImage mergeSourceImageAndCustomImageProperties(
             StatedImage statedImage, CustomImage customImage, String imageCatalogUrl, String catalogName) {
         Image image = statedImage.getImage();
         Image result = new Image(image.getDate(),
+                customImage.getCreated(),
                 customImage.getCreated(),
```

Comment:
is this intentional that created and published timestamp is the same in case of custom image?

Reply:
Yes, it is intentionally the same as when we create a custom image it is going to be published immediately.

Reply:
ok, makes sense